urth_dash_js/notebook/dashboard-view/polymer-support.js

```diff
@@ -29,7 +29,8 @@ define([
         $(cell.element).on('iron-request-resize-notifications', function(event) {
             console.log('Polymer element requesting resize notifications');
 
-            var target = event.path ? event.path[0] : event.target;
+            // find the "real" event target, which may be in shadow DOM
+            var target = Polymer.dom(event.originalEvent).rootTarget;
```

Comment:
Not sure if this helps...
https://www.polymer-project.org/1.0/docs/devguide/events.html#retargeting

You can send the event to Polymer.dom() to get a normalized event.

On Fri, Oct 30, 2015 at 10:42 AM, Javier Pedemonte <notifications@github.com
> wrote:

> In urth_dash_js/notebook/dashboard-view/polymer-support.js
> <https://github.com/jupyter-incubator/dashboards/pull/82#discussion_r43516780>
> :
>
> > @@ -29,7 +29,8 @@ define([
> >          $(cell.element).on('iron-request-resize-notifications', function(event) {
> >              console.log('Polymer element requesting resize notifications');
> >
> > -            var target = event.path ? event.path[0] : event.target;
> > +            // find the "real" event target, which may be in shadow DOM
> > +            var target = Polymer.dom(event.originalEvent).rootTarget;
>
> @lbustelo <https://github.com/lbustelo> @deedubbu
> <https://github.com/deedubbu> I'm a bit hesitant about this change and
> want to get your opinion. Basically, I have one Polymer element
> upcoming-meetups which makes use of another in its template (iron-list).
> When upcoming-meetups loads, iron-list fires an event which we listen
> for. But, when using the original code, event.target would appear as
> upcoming-meetups (I assume since iron-list is within the shadow DOM).
>
> We save event.target for later use, when we fire the iron-resize event.
> But because we saved upcoming-meetups, iron-list never gets this event.
>
> The change above lets us instead save iron-list as the target and
> everything works. But I'm worried that we're saving a reference to an
> element within shadow DOM. Does this seem correct to you?
>
> The other option, I guess, would be to have upcoming-meetups pass events
> down into its descendants. Gave that a quick shot yesterday but failed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter-incubator/dashboards/pull/82/files#r43516780>.
>